packages/components/rating/src/Rating.tsx

```diff
@@ -105,6 +106,16 @@ export const Rating = forwardRef<HTMLDivElement, RatingProps>(
       setRatingValue(Number(event.target.value))
     }
 
+    function onInpuKeyDown(event: KeyboardEvent<HTMLInputElement>) {
```

Comment:
Instead of adding this new handler, I think we could just bail out from the `onInputChange` if `event.target.value === '0'`

Reply:
Not suze, as we need to still accept a 0 value at the beginning. We just want to prevent user from decreasing value to 0.

Reply:
Did you try it? 😉

Reply:
I hate when you're right... :disappointed: Thx :+1: